src/CacheWarmer/CompileCacheWarmer.php

```diff
@@ -38,14 +38,16 @@ public function isOptional()
      */
     public function warmUp($cacheDir): void
     {
-        if ($this->compiled) {
-            // use warm up cache dir if type generator cache dir not already explicitly declare
-            $baseCacheDir = $this->typeGenerator->getBaseCacheDir();
-            if (null === $this->typeGenerator->getCacheDir(false)) {
-                $this->typeGenerator->setBaseCacheDir($cacheDir);
-            }
-            $this->typeGenerator->compile(TypeGenerator::MODE_WRITE | TypeGenerator::MODE_OVERRIDE);
-            $this->typeGenerator->setBaseCacheDir($baseCacheDir);
+        if (!$this->compiled) {
+          return;
```

Comment:
```suggestion
            return;
```

Fix CS: add 2 more spaces